src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.windows.cs

```diff
@@ -38,27 +38,34 @@ public partial class Snackbar
 	{
 		await DismissNative(token);
 		token.ThrowIfCancellationRequested();
+#if WINDOWS10_0_18362_0_OR_GREATER
```

Comment:
as I understand, we set the min version in csproj file, so this check is ignored.
the next "if" doesn't influence the warning.
that is why we still need pragma.

Do you know some other workarounds?

Reply:
NOT RELATED TO PREVIEW 14, but fixes issue on Windows 17763